modules/search/class.jetpack-search.php

```diff
@@ -1119,6 +1138,15 @@ public function convert_wp_es_to_es_args( array $args ) {
 			unset( $es_query_args['sort'] );
 		}
 
+		if ( ! empty( $filters ) && is_array( $filters ) ) {
```

Comment:
Hmm, not sure about this block - needs investigation.

Reply:
Yeah this block is wrong - it was removed in https://github.com/Automattic/jetpack/commit/84c39273984a02f66a84127f9d72fad222ade929#diff-47deebe9b1fd7090caed8291af235f54 after refactoring